src/analysis.cpp

```diff
@@ -1311,40 +1311,60 @@ void ScatteringFunction::_sample() {
     for (int id : ids) { // loop over molecule names
         auto groups = spc.findMolecules(id);
         for (auto &g : groups) // loop over groups
-            if (usecom && !g.atomic)
+            if (use_com && !g.atomic)
```

Comment:
Add one more check that the group is active for use with GCMC, typically by requiring that `g.size()==g.capacity()`

Reply:
Actually, `findMolecules()` already by default set's `ACTIVE`, so never mind.